packages/components/src/DateTimePickers/subpickers/DatePicker/DayPickerAction/DayPickerAction.scss

```diff
@@ -0,0 +1,26 @@
+@import '../../../shared/common';
+
+.action {
+    background: none;
+    border: none;
+    width: 2em;
+    height: 2em;
+    border-radius: 50%;
+
+    &:hover {
+        background-color: $concrete;
+    }
+
+    &:disabled {
+        color: $silver-chalice;
+        background-color: transparent;
```

Comment:
Put that in `.action`, so non disabled days benefit too

Reply:
If i do that, I'll have a background when hovering on disabled action. It was here to avoid that.

Reply:
&#10005;